### PR TITLE
Create 1:9.20220322-4.19.95+revpi1 release [REVPI-1939]

### PR DESCRIPTION
**Pull request by iluminat23**
Signed-off-by: Philipp Rosenberger <p.rosenberger@kunbus.com>

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,18 @@
+raspberrypi-firmware (1:9.20220322-4.19.95+revpi1) stable; urgency=medium
+
+  * dts: revpi-core: pull down sniff pin
+  * piControl: add module depency for DIN on compact
+  * piControl: use correct function to exchange data with userspace
+  * Use hardware interrupt timer instead in soft interrupt
+  * compact: Shrink critical section (cpu temp/freq)
+  * flat: Shrink critical region in the ain thread
+  * fix: The removal of elevator= from the cmdline.txt is not working
```
Comment (nbuchwitz) [inline debian/changelog:9]: Not really related to this PR, but It raised my attention: Will this commit break something with 4.19? Originally this was intended for 5.10

**Review by nbuchwitz — CHANGES_REQUESTED** [review]

File: debian/changelog (hunk 1 above)
Comment (nbuchwitz) [reply debian/changelog:9]: (line 9, elevator)

Comment (iluminat23) [reply debian/changelog:9]: If I remember it correctly, I tested this with 4.19 and the default elevator is already deadline. So it shouldn't change anything. I think I have written this in the ticket.
I check if I find it ...

Comment (nbuchwitz) [reply debian/changelog:9]: Found it: "The deadline scheduler is also the default on 4.19" (REVPI-1647)

**Review by nbuchwitz — APPROVED** [review]
LGTM

**Review by linosanfilippo-kunbus — APPROVED** [review]

**Pull request merged**